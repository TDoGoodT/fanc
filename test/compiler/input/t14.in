int decrement(int num) {
    return num - 1;
}
bool isEven(int num) {
    if (num / 2 == (num + 1) / 2) {
        return true;
    } else {
        return false;
    }
}

void main() {
    int num1 = 6;
    int num2 = 7;

    bool result1 = isEven(num1);
    bool result2 = isEven(num2);

    if (result1) {
        print("6 is even");
    } else {
        print("6 is odd");
    }

    if (result2) {
        print("7 is even");
    } else {
        print("7 is odd");
    }
}
