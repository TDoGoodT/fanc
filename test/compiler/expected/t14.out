6 is even
7 is odd
